app/views/user_mailer/invitation.html.erb

```diff
@@ -12,18 +12,22 @@
   </p>
 
   <p>
-    <%= t ".part1" %>
+    <%= t(".html.about", otw_link: style_link(t(".html.otw_link_text"), "https://transformativeworks.org")).html_safe %>
   </p>
 
   <p>
-    <%= t ".part2" %>
+    <%= t ".features" %>
   </p>
 
   <p>
-    <%= t(".html.part3", invitation_link: style_link(signup_url(:invitation_token => @invitation.token), signup_url(:invitation_token => @invitation.token))).html_safe %>
+    <%= t(".html.join", invitation_link: style_link(t(".html.invitation_link_text"), signup_url(:invitation_token => @invitation.token))).html_safe %>
```

Comment:
Could you update the syntax here (and in the text email) as described in [this comment](https://github.com/otwcode/otwarchive/pull/4268#discussion_r904203587)? Then I think this will be done!

Reply:
done :)